_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1066,6 +1075,95 @@ private function _process_onboarding( $data ) {
 			: join( ', ', $error );
 	}
 
+	/**
+	 * Retrieve the first active sidebar.
+	 *
+	 * @return array|WP_Error First active sidebar, error if none exists.
+	*/
+	static function get_first_sidebar() {
```

Comment:
How about moving this one to the `widgets` lib too? It's generic and not directly connected to the business address logic.

Reply:
yes, missed that one 👍